docs/reference/installation.rst

```diff
@@ -8,7 +8,7 @@ Installation
 Prerequisites
 -------------
 
-PHP ^7.2 and Symfony ^4.4 are needed to make this bundle work, there are
+PHP ^7.4 and Symfony ^4.4 are needed to make this bundle work, there are
```

Comment:
On other projects we are removing the php and symfony requirements from here, like media bundle. You can take a look there for reference

Reply:
Done